frontend/styles/how-it-work.css

```diff
@@ -0,0 +1,5 @@
+.num-component > div {
```

Comment:
Can't you get rid of this css file and use tailwind instead?!

Reply:
( PersonCard ) Component doesn't accept className property

Reply:
you can wrap it with a div that takes the style or you can change the person card component itself to accept custom classes (I suggest the second choice)